Physicc/include/narrowphase.ipp

```diff
@@ -21,6 +23,36 @@ bool checkCollision<Box, Sphere>(Broadphase::PotentialContact a)
 	// For BoxSphere, the first is a box, the second a sphere.
 	// Returns true if the box and the sphere in `a` are colliding.
 	// Otherwise returns false. Also writes the data to the public collision array.
+	
+	auto box = a.rb1;
+	auto sphere = a.rb2;
+
+	glm::vec3 s_centre = sphere.getCentroid();
```

Comment:
Given that `s_` usually indicates a static variable, perhaps this is not the best choice for variable names. Also, we use the camelCase naming case, so I would name it `sphereCenter`.

Reply:
Made the requested changes.